pkg/apis/nodeup/config.go

```diff
@@ -73,6 +73,9 @@ type Config struct {
 	ConfigServer *ConfigServerOptions `json:"configServer,omitempty"`
 	// AuxConfigHash holds a secure hash of the nodeup.AuxConfig.
 	AuxConfigHash string
+
+	// Containerd config holds the configuration for containerd
+	Containerd *ContainerdConfig `json:"containerd,omitempty"`
```

Comment:
Can be extended later if there's any need for it.
```suggestion
	ContainerdConfig string `json:",omitempty"`
```

Reply:
This config is unversioned. I don't think we can extend it in the future as that would break nodeup/cloudup marshaling.

Reply:
But it's just part of the launch template, which is updated together with new version of nodeup?

Reply:
If I understand @johngmyers correctly, there is an idea about writing this to VFS at some point.

Reply:
That is an alternative for large user data cases, if I understand correctly.
For containerd there are no flags to control various aspects, everything can be done through the config file, which will be generated in cloudup. So, I don't see any need for extending this short term.

Reply:
The fact that it's unversioned isn't a problem because the reader (nodeup) always matches the version of the writer (kops). So we can make incompatible changes with impunity.